Doc/library/sqlite3.rst

```diff
@@ -2269,8 +2339,67 @@ Explanation
 Transaction control
 ^^^^^^^^^^^^^^^^^^^
 
-The :mod:`!sqlite3` module does not adhere to the transaction handling recommended
-by :pep:`249`.
+.. _sqlite3-transaction-control-autocommit:
+
+Transaction control via the ``autocommit`` attribute
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The recommended way of controlling transaction behaviour is through
+the :attr:`Connection.autocommit` attribute.
+This attribute should preferrably be set using the *autocommit* parameter
+of :func:`connect`.
+
+It is recommended to set *autocommit* to ``False``,
+implying :pep:`249`-compliant transaction control.
+This means:
+
+* :mod:`!sqlite3` ensures that a transaction is always open,
+  so :meth:`~Connection.commit` and :meth:`~Connection.rollback`
+  will implicitly open a new transaction immediately after closing
+  the pending transaction.
+  :mod:`!sqlite3` uses ``BEGIN DEFERRED`` statements when opening transactions.
+* Transactions should be committed explicitly
+  using :meth:`!commit`.
+* Transactions should be rolled back explicitly
+  using :meth:`!rollback`.
+* An implicit rollback is performed if the database is
+  :meth:`~Connection.close`-ed with pending changes.
+
+Set *autocommit* to ``True`` to enable SQLite's `autocommit mode`_.
+In this mode, :meth:`~Connection.commit()` and :meth:`~Connection.rollback()`
```

Comment:
```suggestion
In this mode, :meth:`Connection.commit` and :meth:`Connection.rollback`
```

Again, best to be explicit I think, and the `()`s are unnecessary (as they are handled automatically)

Reply:
Yeah, I'm not sure why the parentheses were there. Leftovers from a very early revision I guess.